bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/dialogs/CheckedTreeSelectionDialog.java

```diff
@@ -228,6 +245,16 @@ public void setSize(int width, int height) {
 		fHeight = height;
 	}
 
+	/**
+	 * Returns whether an empty tree is allowed
+	 *
+	 * @return true if empty tree is ok, false otherwise
+	 * @since 3.131
+	 */
+	protected boolean isEmptyTreeAllowed() {
```

Comment:
Is this method actually consumed in any current client? ie, could it be removed?